src/queue.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""Create a Queue object that fits the definition defined here:
+    http://www.princeton.edu/~achaney/tmve/wiki100k/docs/Queue_(data_structure).html
+    http://ls.pwd.io/2014/08/singly-and-doubly-linked-lists-in-python/
+    It should have the following methods:
+
+    .enqueue(value): Add a value to the queue
+    .dequeue(value): Remove a value from the queue and return the value
+        - If empty, raise an Exception
+    .peek(): Return the next value
+        - If empty, return None
+    .size(): Check the size of the queue
+
+    This queue follows a logic of first in, first out.
+"""
+
+
+class Node(object):
+    """Classify a NodeObject that points to two different objects."""
+
+    def __init__(self, value, next, prev):
+        """Initialize node."""
+        self.value = value
+        self.next = next
+        self.prev = prev
+
+
+class Queue(object):
+    """Classify a new object that resembles a LinkedList following a first in first out logic."""
```

Comment:
[PEP 257](https://www.python.org/dev/peps/pep-0257/) for docstrings in this class.